File: tys-spa/ciudadano/src/app/domain/datos-empresa.ts
import {Empresa} from './empresa';
import {Persona} from './persona';
import {SocioEmpresa} from './socio-empresa';
import {Direccion} from './direccion';

export interface DatosEmpresa {
  empresa: Empresa;
  listaPersonas?: Array<Persona>;
  listaSocios?: Array<SocioEmpresa>;
  direccionSucursal?: Direccion;
  datosEmpleador?: any;
}

File: tys-application/tys-commons/src/main/java/co/com/mintrabajo/tys/commons/domain/DatosEmpresa.java
package co.com.mintrabajo.tys.commons.domain;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
public class DatosEmpresa  {

	private Empresa empresa;
	private List<Persona> listaPersonas;
	private List<SocioEmpresa> listaSocios;
	private Direccion direccionSucursal;
	private List <Direccion> listaDireccionesSucursales;
	private DatosEmpleador datosEmpleador;
	public DatosEmpresa(){
		
	}

}
